src/components/layout/FooterSection.vue

```diff
@@ -61,7 +61,7 @@
       class="login-footer"
     >
       <OutlinedButton
-        :href="addressDeepLink"
+        :href="addressDeepLink.toString()"
```

Comment:
Technically, we can make it accept Url instance as well, but that component anyway is not enought flexible now, so I don't know how to handle it better. The best would be to make all buttons to accept full url, router location, click handler (displaying it as <a/button/RouterLink>) as we did in the [Base aepp](https://github.com/aeternity/aepp-base/blob/d17541574773e0b2db3fd1b66b1b0a4ea06ccf4a/src/components/ButtonPlain.vue).